util/include/Metrics.hpp

```diff
@@ -70,6 +70,8 @@ class Gauge {
   explicit Gauge(const uint64_t val) : val_(val) {}
 
   void Set(const uint64_t val) { val_ = val; }
+  void Inc(const uint64_t val) { val_ += val; }
+  void Dec(const uint64_t val) { val_ -= val; }
```

Comment:
Counters should monotonically grow. Why do you need to decrement a counter?

Reply:
This is identical to the Prometheus approach (see https://github.com/jupp0r/prometheus-cpp/blob/master/core/include/prometheus/gauge.h).

It helps to have a ++ like operation for gauges.

Reply:
Anyway, I noticed that I'm not really using it, so it has been removed